packages/cli/src/databases/entities/User.ts

```diff
@@ -103,6 +103,15 @@ export class User extends AbstractEntity implements IUser {
 	@Index({ unique: true })
 	apiKey?: string | null;
 
+	@Column({ type: String, nullable: true })
```

Comment:
an issue with adding these columns on the user table is that now every query for a user will include this data. 
I'm not sure what a better alternative would be, just want to make sure that this gets discussed before we merge these migrations.

Reply:
@netroy, what about setting mfaSecret and mfaRecoveryCodes as `select: false`, and querying them in the MFA endpoints only, which is the only place where mfaSecret and mfaRecoveryCodes are needed?

> select: boolean - Defines whether or not to hide this column by default when making queries. When set to false, the column data will not show with a standard query. By default column is select: true

Reply:
@netroy what do you think?

Reply:
I'd be okay with that for now. but, I'm thinking that in the long term, we need to split the user table into 2 tables, one for the columns that are used in the app, and another for everything related to auth, that would eventually be only used by an auth service that optionally lives outside the core code. Considering the size of the users table, I think we can handle that split via a migration.